src/routes/safe/components/Apps/hooks/useIframeMessageHandler.ts

```diff
@@ -58,6 +49,7 @@ const useIframeMessageHandler = (
       const requestWithMessage = {
         ...message,
         requestId: requestId || Math.trunc(window.performance.now()),
+        version: '0.4.2',
```

Comment:
is this fixed for any reason? shouldn't we take it from package.json?

Reply:
we are not going to update 0.x.x sdk